api/envoy/config/bootstrap/v3/bootstrap.proto

```diff
@@ -447,6 +448,17 @@ message Admin {
   // Indicates whether :ref:`global_downstream_max_connections <config_overload_manager_limiting_connections>`
   // should apply to the admin interface or not.
   bool ignore_global_conn_limit = 6;
+
+  // Optional list of routes accessible to the admin. The allowed routes can be matched by
+  // any patterns defined in the StringMatcher proto (e.g. exact string, prefix, regex, etc).
+  // If not specified, the allowlist check will be skipped.
+  //
+  // Example:
+  // allow_interfaces:
+  // - prefix: foo
+  // - exact: foo
+  // - exact: bar
+  repeated type.matcher.v3.StringMatcher allow_interfaces = 7;
```

Comment:
i see the comment by @wbpcode but i am not sure if "interfaces" is the right word. This is literally a list of http request paths/routes and I think either paths or routes is much clearer?

Reply:
This mainly to ensure the API is consistant with our docs. I am fine to change both but will not suggest to do that because the the `Administration interface` is there for a very very long time.